index.Rmd

```diff
@@ -31,22 +31,20 @@ Raster data is like any image. Although it may portray various properties of obj
 
 Today's tutorial is about constructing a simple spatio-temporal analysis using raster data, R, and git.
 
-## Assumed knowledge from previous tutorials
-
-* Understand system architecture (R, R packages, libraries, drivers, bindings)
-* Good scripting habits
-
-
-# Reminder on overall system architecture
+# The overall system architecture
```

Comment:
Correct me if I'm wrong, but I don't think the first point of assumed knowledge has been covered in previous tutorials.

Reply:
You're right!